src/core/services/ripple/ripple.js

```diff
@@ -449,7 +552,9 @@ InkRippleCtrl.prototype.removeRipple = function (ripple) {
  */
 InkRippleCtrl.prototype.fadeOutComplete = function (ripple) {
   ripple.remove();
-  this.lastRipple = null;
+  if (this.lastRipple === ripple){
```

Comment:
Nitpicky again: Please space the brace here, just to be consistent with the Code Conventions.